crates/biome_cli/src/execute/migrate/prettier.rs

```diff
@@ -342,6 +344,11 @@ pub(crate) fn read_prettier_files(
         }));
     } else {
         let prettier_configuration = deserialized.into_deserialized();
+        if prettier_configuration.clone().unwrap().end_of_line == EndOfLine::Auto {
+            console.log(markup! {
+                <Warn>"Warning: Prettier's `\"endOfLine\": \"auto\"` option is not supported in Biome. Using the default 'lf' option."</Warn>
+            });
+        }
 
         if let Some(prettier_configuration) = prettier_configuration {
```

Comment:
You can put the `if` check inside the `if let` block, and you don't need `clone` or `unwrap` anymore.

```suggestion
        if let Some(prettier_configuration) = prettier_configuration {
          if prettier_configuration.end_of_line == EndOfLine::Auto {
              console.log(markup! {
                  <Warn>"Warning: Prettier's `\"endOfLine\": \"auto\"` option is not supported in Biome. Using the default 'lf' option."</Warn>
              });
          }
```

Reply:
thanks 👍  updated